packages/ffe-cards/less/icon-card.less

```diff
@@ -10,14 +10,30 @@
 
     display: grid;
     grid-template-columns: auto 1fr;
+    grid-template-areas: "icon body";
     align-items: center;
     padding: var(--ffe-spacing-md);
 
     & > &__icon {
+        grid-area: icon;
         color: var(--ffe-v-cards-icon-color);
         margin: 0 var(--ffe-spacing-lg) 0 var(--ffe-spacing-xs);
     }
 
+
```

Comment:
Jag er skeptisk tip att endre rekkefolge kun med css pga av att man ikke vet vad folk stopper in som icon. Hvis det er ett element som kan ha focus så ødelgger man focus rekkefolgen og det blir en dålig opplevelse før tastaturbruker da focus hoper omkring ologisk. Vi kan bruke css men endre gjerne også rekkefolgen på html i kortet.